app/javascript/dashboard/routes/dashboard/settings/inbox/channels/Website.vue

```diff
@@ -153,6 +171,18 @@ export default {
     ...mapGetters({
       uiFlags: 'inboxes/getUIFlags',
     }),
+    textAreaChannels() {
+      if (
+        this.isATwilioChannel ||
+        this.isATwilioSMSChannel ||
+        this.isATwilioWhatsappChannel ||
+        this.isATwitterInbox ||
+        this.isFacebookChannel ||
+        this.isAFacebookInbox
```

Comment:
Why do we need to check both `isFacebookChannel` and `isAFacebookInbox`? Also, I don't think `isFacebookChannel` is defined.

We can omit the check for WhatsApp and SMS, check for twilio is enough.

Reply:
Removed unnecessary check in https://github.com/chatwoot/chatwoot/pull/2952/commits/69a930ed1b8ac310a2eb2633504afbb417f5398a